partiql-planner/src/main/kotlin/org/partiql/planner/internal/PlanningProblemDetails.kt

```diff
@@ -166,6 +166,13 @@ internal open class PlanningProblemDetails(
         "Unknown function `$identifier($types)"
     })
 
+    data class UnknownCast(
```

Comment:
```suggestion
    public data class UnknownCast(
```

Reply:
This is embedded within the internal `PlanningProblemDetails`.